flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/test/java/org/apache/flink/cdc/connectors/paimon/sink/PaimonMetadataApplierTest.java

```diff
@@ -416,4 +416,87 @@ public void testAddColumnWithPosition(String metastore)
         Assertions.assertEquals(
                 tableSchema, catalog.getTable(Identifier.fromString("test.table1")).rowType());
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"filesystem", "hive"})
+    public void testOfflineAlterExistsTableSchema(String metastore)
```

Comment:
So the expected scenario to be solved is actually a scenario where the table structures of downstream that was previously created are inconsistent with upstream(mysql)?

Reply:
Yes

Reply:
I think this change carries risks, I tend to add a sink option to explicitly support this matter:
When the upstream and downstream table structures are inconsistent, users can manually modify the table structure to adapt.
If this is done in the program, in the case where the user configures the wrong `route` rule (which is not a particularly rare situation), the situation where the table structure does not correspond does not throw an exception. We will forcibly modify downstream tables to adapt, and the user will not be aware of this error.

Reply:
thanks, You are right, I only considered the \* situation before

Reply:
@lvyanquan Could you please review it again? thank you